starfish/intensity_table.py

```diff
@@ -147,18 +160,23 @@ def from_spot_data(
                 f'shape ({intensities.shape}) is invalid.')
 
         cls._verify_spot_attributes(spot_attributes)
+        coords = cls._build_coordinates(
+            spot_attributes,
+            np.arange(intensities.shape[1]),
+            np.arange(intensities.shape[2]))
 
-        coords = (
-            (Features.AXIS, spot_attributes),
-            (Indices.CH.value, np.arange(intensities.shape[1])),
-            (Indices.ROUND.value, np.arange(intensities.shape[2]))
-        )
+        # coords = (
```

Comment:
did you mean to leave these comments in?

Reply:
No, sorry. 😞